build.gradle

```diff
@@ -64,7 +64,8 @@ javadocJar.enabled = false
 ext {
     slf4jVersion = '1.7.7'
     servletApiVersion = '3.1.0'
-    zipkinVersion = '1.7.0'
+    zipkinCoreVersion = '1.7.0'
+    zipkinReporterVersion = '0.2.0'
```

Comment:
I changed my mind - I think I'll decline this PR for now and give you time to get the reporter libs into a more stable state. The old copy-paste code works fine for now, and I'll keep this commit around. Should be easy to update to whatever you end up with when the time comes.